pkg/util/logging/default_logger.go

```diff
@@ -31,11 +34,31 @@ func DefaultHooks() []logrus.Hook {
 	}
 }
 
+type LogFormatOptions struct {
```

Comment:
Let's call this `FormatFlag` (so it'll be referenced from other packages as `logging.FormatFlag`)